enclave-runtime/src/tests.rs

```diff
@@ -433,7 +406,13 @@ fn test_create_block_and_confirmation_works() {
 
 	// when
 	let (confirm_calls, signed_blocks) =
-		crate::execute_top_pool_calls(&OcallApi, &top_pool, latest_onchain_header).unwrap();
+		crate::exec_tops_for_all_shards::<Block, SignedBlock, _, _>(
+			&OcallApi,
+			&top_pool,
+			&latest_onchain_header,
+			GETTER_TIMEOUT + CALL_TIMEOUT,
```

Comment:
if you use this combination multiple times, wouldn't it make sense to define a new variable to represent to aggregation of these 2 values?

Reply:
Hmm, good catch. I didn't bother too much, but because I thought they will be remove when `Aura` is introduced. But that is actually not true for these tests.